### DISCUSSION TITLE
revalidate all fields when some field triggers onBlur

### DISCUSSION THREAD

**Discussion by pavelspichonak** [react-hook-form/react-hook-form#4045] [Q&A]
Hi @bluebill1049 

I'm using default `mode` and `reValidateMode: 'onBlur'`. But `onBlur` triggers revalidation only for that field which triggers `onBlur` event.

There are cases when some field triggers `onBlur` you need to revalidate all fields. For example you have two fields `password` and `confirmPassword`. `confirmPassword` field has validation based on `password` field value. So when `password` field triggers `onBlur` event and revalidated, the `confirmPassword` field should be revalidated as well.

How can I configure `reValidateMode` to be able to revalidate all fields when some field triggers `onBlur`?

I know about `trigger` method. But how can I call it if any field in the form triggers `onBlur`? It is not good to subscribe manually to each field in a form like this `onBlur={() => triggers()}`. There should be some option I do not know.

Any ideas? Thanks.

**Comment by bluebill1049** [comment]
we don't have such mechanism.

**Reply by pavelspichonak** [reply]
Maybe you can add such mechanism?

**Reply by pavelspichonak** [reply]
For example `reValidateMode: 'onBlurAll'`
